clients/python/test/e2e/test_notebooks.py

```diff
@@ -45,3 +46,90 @@ def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any] =
         kernel_name="python3",
         parameters=params,
     )
+
+
+def get_tutorials(osparc_version: Optional[str] = None) -> List[Path]:
```

Comment:
Q: `Optional` or `Dict` annotations are because we have to be compatible with 3.6>?